src/ReactLeafletMap/index.js

```diff
@@ -256,6 +256,7 @@ class ReactLeafletMap extends React.PureComponent {
                 key={layer.key || i}
                 url={o.url}
                 crs={crs}
+                singleTile={o.singleTile === true}
```

Comment:
```suggestion
                singleTile={o.singleTile}
```

Reply:
o.singleTile is optional, so checking o.singleTile === true secure, that singleTile props will be always bool.

In case you suggest, singleTile could be undefined

Reply:
If undefined, then falsy. Isn't it enough?

Reply:
Which one?

Reply:
?...In my opinion, the type shoud be checked via Prop-Types.

Reply:
We do not have typechecking of options. I'm not sure if we are talking about same thinks. 

Yes ```singleTile={o.singleTile}``` could work, but ``` singleTile={o.singleTile === true}``` is better from my point of view.

https://github.com/gisat-panther/ptr-maps/pull/30/files#diff-9ef14713dda97d138a2f82c7ace459d2R9